pinakes/main/approval/tests/functional/test_notification_end_points.py

```diff
@@ -0,0 +1,56 @@
+""" Module for testing Notification settings """
+import pytest
+from pinakes.main.approval.models import Notification
+from pinakes.main.approval.permissions import TemplatePermission
+
+
+@pytest.mark.django_db
+def test_notification_list(api_request, mocker):
+    """Get a list of templates"""
+    has_permission = mocker.spy(TemplatePermission, "has_permission")
+    response = api_request("get", "approval:notification-list")
+
+    assert response.status_code == 200
+    assert response.data["count"] == 1  # seeded one
+    has_permission.assert_called_once()
+
+
+@pytest.mark.django_db
+def test_notification_retrieve(api_request, mocker):
+    """RETRIEVE a template by its id"""
+    has_permission = mocker.spy(TemplatePermission, "has_permission")
+    notification = Notification.objects.first()
+    response = api_request(
+        "get", "approval:notification-detail", notification.id
+    )
+
+    assert response.status_code == 200
+    assert response.data["id"] == notification.id
+    has_permission.assert_called_once()
+
+
+@pytest.mark.django_db
+def test_notification_not_supported_methods(api_request):
```

Comment:
@bzwei Will you support these APIs in the future? Users will need them to configure their notification settings, right?

Reply:
Notification is the schema for settings. User will not be able to change the schema. The actual settings are stored in Template.